torchtune/modules/transforms/_transforms.py

```diff
@@ -127,16 +127,15 @@ def __call__(self, sample: Mapping[str, Any]) -> Mapping[str, Any]:
                     (n_tiles, c, h, w) each.
 
         Returns:
-            Mapping[str, Any]: updated sample with the following keys:
-                - encoder_mask (List[torch.Tensor]): list of masks with shape (text_seq_len, image_seq_len),
-                    where length of list == number of images in sample
-                - tokens (List[int]): original tokens
-                - images (List[torch.Tensor]): original images
+            Mapping[str, Any]: sample with a new key encoder_mask, with a mask per image with shape
```

Comment:
why are we packaging multiple keys into encoder_input? also, you mention encoder_mask here but use encoder_input below. Would also be good to keep the bullets about tokens and images